src/wx/CMakeLists.txt

```diff
@@ -89,6 +89,11 @@ if(NOT WIN32 AND NOT APPLE)
     endif()
 endif()
 
+# Win32 definitions common to all toolchains.
+if (WIN32)
+    add_definitions(-D_UNICODE -DUNICODE -DwxUSE_GUI=1 -D__WXMSW__ -DWINVER=0x0A00 -DNTDDI_VERSION=0x0A000007)
+endif()
```

Comment:
I gather this sets compatibility to win10. Do we really need this for the features that we need? Could we set compatibility to 7 or even Vista?

Reply:
This is confusing but `WINVER` and `NTDDI_VERSION` are used by the Windows SDK as an indicator for "I want to access APIs for Windows version up to X". There are ways to check at runtime if an API is present or not. Anyway, simply defining them does not result in Windows-10-only binaries, it merely gives access to Windows 10 APIs.
We do need to be careful when we use Windows APIs to provide fallback code paths for older versions but this is an orthogonal issue.